test/e2e/scheduling/preemption.go

```diff
@@ -55,6 +55,10 @@ type priorityPair struct {
 	value int32
 }
 
+// construct a fakecpu so as to set it to status of Node object
+// otherwise if we update CPU/Memory/etc, those values will be corrected back by kubelet
+var fakecpu v1.ResourceName = "example.com/fakecpu"
```

Comment:
LGTM generally, can you also fold this into the `ginkgo.Context("PreemptionExecutionPath", func()` section?

Reply:
Ping @Pulkit07

Reply:
I folded this, thanks for review.